iOS/DuckDuckGo/AIChat/InputBox/SwitchBar/OmniBarEditingStateViewController.swift

```diff
@@ -467,3 +468,37 @@ extension OmniBarEditingStateViewController {
         )
     }
 }
+
+// MARK: - Extract Suggestions
+extension OmniBarEditingStateViewController {
+    private func extractText(from suggestion: Suggestion) -> String? {
```

Comment:
Would be good to extract this to a common entity and use it in both here and `MainViewController`.

Reply:
Yep, added to the [list](https://app.asana.com/1/137249556945/project/72649045549333/task/1210716458597585?focus=true)